src/components/Exercise/styles.ts

```diff
@@ -1,11 +1,24 @@
 import { css } from "styled-components";
 
 export const exerciseStyles = css`
+
+  &.included-card {
+    background-color: #daf3f8;
```

Comment:
You can use the color for this in the theme file, though you may want to abstract the name a little so it's not awkward using a "header" key here.